python/tabby/tabby-data-pipeline/modal/predict.py

```diff
@@ -62,16 +65,17 @@ def __enter__(self):
 
         my_env = os.environ.copy()
         my_env["TABBY_DISABLE_USAGE_COLLECTION"] = "1"
-        MODEL_ID = os.popen("cat /tmp/tabby_model_id").read().strip()
+        MODEL_ID = os.environ.get("MODEL_ID")
         print(f'MODEL_ID={MODEL_ID}')
+       
         LAUNCH_FLAGS = ["serve", "--model", MODEL_ID, "--port", "8000", "--device", "cuda"]
         self.launcher = subprocess.Popen(["/opt/tabby/bin/tabby"] + LAUNCH_FLAGS, env=my_env)
         self.client = Client("http://127.0.0.1:8000", timeout=240)
 
         # Poll until webserver at 127.0.0.1:8000 accepts connections before running inputs.
         def webserver_ready():
             try:
-                socket.create_connection(("127.0.0.1", 8000), timeout=1).close()
+                socket.create_connection(("127.0.0.1", 8000), timeout=30).close()
```

Comment:
```suggestion
                socket.create_connection(("127.0.0.1", 8000), timeout=1).close()
```

Line 91 already contains retry logic, no need to increase timeout